security/firely-auth/firely-auth-endpoints.rst

```diff
@@ -117,3 +117,31 @@ Known Limitations
 
 * In Firely Auth no Backchannel Authentication Endpoint is available, therefore Client Initiated Backchannel Authentication (CIBA) requests are not supported. For more information, see `Duende Documentation - Client Initiated Backchannel Authentication (CIBA) <https://docs.duendesoftware.com/identityserver/v6/reference/endpoints/ciba/>`_.
 * A Device Authorization Flow is not supported by SMART on FHIR. Therefore it is not available in Firely Auth. For more information, see `Duende Documentation - Device Authorization Endpoint <https://docs.duendesoftware.com/identityserver/v6/reference/endpoints/device_authorization/>`_.
+
+Liveness and readiness
+----------------------
+
+It can be useful to check whether Firely Auth is still up and running, and ready to handle requests. Either just for notification, or for automatic failover.
+A prominent use case is to start dependent services only after Firely Auth is up and running, e.g. in a docker-compose or in a Helm chart.
+
+Firely Auth provides two endpoints, for different purposes:
+
+* ``GET <base>/$liveness``
+* ``GET <base>/$readiness``
+
+These align - intentionally - with the use of liveness and readiness probes in Kubernetes.
+
+Results
+-------
+
+The ``$liveness`` operation may return one of these http status codes:
+
+#. 200 OK: Firely Server is up and running (but may still be blocked by a long running process).
```

Comment:
@rgrotenbreg see the comment in  https://firely.atlassian.net/browse/VONK-7337?focusedCommentId=55196
We don't really return anything other than a 200 OK response, so I am not sure if I would mention the other response types here.

Reply:
we return 200, 402 and a component in front of FA could return a 500, like a load balancer, updated as there will be no long running process